### PR TITLE
check_deprecation_warnings: improve error message

### PR DESCRIPTION
**Pull request by jeremylenz**
Addresses a couple issues I had running `check_deprecation_warnings` locally:

* Improves error messaging if it can't find the repo path.
* Fixes a small bug where it was trying to run a regex `.scan` on something that was expected to be a String, but was not.

### CODE REVIEW

File: lib/tool_belt/commands/check_deprecation_warnings.rb [hunk 1]
```diff
@@ -10,7 +10,9 @@ def execute
         config = ToolBelt::Config.new(config_file)
         options = config.options
         repo_path = "repos/#{options.namespace}/#{options.project}/"
-        raise "Repo path #{repo_path} not found!" unless File.exist?(repo_path)
+        unless File.exist?(repo_path)
+          raise "Repo path #{repo_path} not found! Did you forget to run './tools.rb setup-environment #{repo_path}' ?"
```
Comment (johnpmitsch) [inline lib/tool_belt/commands/check_deprecation_warnings.rb]: `setup-environment` takes a config file, not a repo path. You could specify `config_file` here or keep it generic saying something like "for this release"

**Comment by johnpmitsch** [comment]
Overall looks good and I like the better error message, just a note on the command specified in the message.

File: lib/tool_belt/commands/check_deprecation_warnings.rb (hunk 1 above)
Comment (jeremylenz) [reply lib/tool_belt/commands/check_deprecation_warnings.rb]: @johnpmitsch Thanks, updated!

**Comment by johnpmitsch** [comment]
@jeremylenz I don't have merge permissions, so you will need someone else to do the honors!

**Pull request merged**